.gitignore

```diff
@@ -1,2 +1,4 @@
 *.kdev4
 build
+/.vs
+/CMakeFiles
```

Comment:
This one is incorrect and should be deleted.

Reply:
Corrected, replaced with .vscode instead